### PR TITLE
fix: filter out non existing page in related conent component

### PR DESCRIPTION
**Pull request by ostafinskim**
[Zendesk](https://studio24.zendesk.com/agent/tickets/14039) ticket reference

### CODE REVIEW

File: resources/views/includes/structure/related-pages.blade.php [hunk 1]
```diff
@@ -1,12 +1,27 @@
 @if(!empty($records))
-    @section('associated_pages')
+@section('associated_pages')
+<div class="container">
+    <h3>Related to this page</h3>
```
Comment (sean-dunwoody) [inline resources/views/includes/structure/related-pages.blade.php:4]: Hi @ostafinskim ,

It looks like you have duplicated the container element and title. Could you please remove the duplicates?

**Review by sean-dunwoody — CHANGES_REQUESTED** [review]

File: resources/views/includes/structure/related-pages.blade.php (hunk 1 above)
Comment (ostafinskim) [reply resources/views/includes/structure/related-pages.blade.php:8]: Done it.

**Review by sean-dunwoody — APPROVED** [review]

**Comment by portableant** [comment]
Instead of having to do this, why not remove the item from solr manually or make a stubs/pages refresh cronjob like the one referenced here - https://github.com/FitzwilliamMuseum/fitz-main-website/blob/master/app/Console/Kernel.php#L76 These were all works in progress....

**Comment by sean-dunwoody** [comment]
Thanks @portableant , we understand this is really not an ideal way of fixing this. We're removing it like this just for the short term due to time pressures. 

We're in the progress of moving the Solr server to a separate AWS service to allow us to manage it more effectively - we're looking at updating the index inline with Directus alongside that (which will involve creating the cron job necessary to update the stubs and pages at set intervals).

**Pull request merged**